### PR TITLE
Fix invalid botmeta entries

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
Correct paths
Remove trailing whitespace

##### ISSUE TYPE
- Bugfix Pull Request

### CODE REVIEW

**Comment by gundalow** [comment]
Note to reviews use https://github.com/ansible/ansible/pull/67785/files?w=1 which will remove whitespace

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -3518,10 +3518,6 @@ files:
     supershipit: $team_google
     maintainers: $team_google
     migrated_to: google.cloud
-  $plugins/doc_fragments/grafana.py:
```
Comment (Akasurde) [inline .github/BOTMETA.yml:3521]: Any reason why we are removing this?

Comment (gundalow) [reply .github/BOTMETA.yml:3521]: This file was added only to the collection. It never existed in devel.

Comment (Akasurde) [reply .github/BOTMETA.yml:3521]: ohk Thanks for the information.

**Pull request merged**